server/api/contest.js

```diff
@@ -1,4 +1,10 @@
-const { isBefore, isFuture } = require('date-fns');
+/**
+ * Get contest detail
+ * FROM FIREBASE: results, voting dates,
+ * FROM REDDIT: 
```

Comment:
For older contests, it gets the entry details (name, description, image) from reddit. For local submissions, it's all from the DB, so this will probably be removed later.